app/views/matches/index.html.slim

```diff
@@ -7,8 +7,10 @@
   table.match-tbl
     thead
       tr
-        th[colspan="5"]
-          = match['date']
+        - if @date_var != match['date']
+          - @date_var = match['date']
```

Comment:
Condition is for printing the date on each day, only once





On Wed, Apr 6, 2016 at 6:31 PM -0700, "Yu Ming" <notifications@github.com> wrote:





> @@ -7,8 +7,10 @@
>    table.match-tbl
>      thead
>        tr
> -        th[colspan="5"]
> -          = match['date']
> +        - if @date_var != match['date']
> +          - @date_var = match['date']

hmm is this condition necessary? how about just do `@date_var = match['date']` ?

---
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub:
https://github.com/moexmen/foosball-leaderboard/pull/8/files/ec2ee07c3c47915b08f032471926f3f05817a54a#r58808099